package.json

```diff
@@ -19,6 +19,7 @@
   ],
   "license": "OGL-UK-3.0",
```

Comment:
Don't forget to bump the semantic version number

Reply:
bumped